pbj-integration-tests/src/test/java/com/hedera/pbj/intergration/test/SampleFuzzTest.java

```diff
@@ -14,19 +21,32 @@
  * See javadoc for FuzzTest for more details.
  */
 public class SampleFuzzTest {
+    // A percentage threshold for the DESERIALIZATION_FAILED outcomes.
+    private static final double THRESHOLD = 0.5;
```

Comment:
Now you inflict more damage to the payload. Should it affect the threshold?

Reply:
Unfortunately, no, not yet. As you can see in the updated output above, we still encounter cases with the DESERIALIZATION_FAILED cases occurring just a bit above 50%. I'll see if I can improve this in future PRs.